functions/src/utils/api/api.ts

```diff
@@ -0,0 +1,68 @@
+import express from 'express';
+import {
+    create,
+    update,
+    get as dbGet,
+    getAll as dbGetAll,
+    deleteOne as dbDeleteOne
+} from '../firebase';
+
+export const post = async (collection: string, req: any, res: any) => {
+    const data = req.body;
+    const response = await create(collection, data);
+
+    res.status(201);
```

Comment:
These are scattered across the backend, but you should probably use constants for http response codes